docsteady/vcd.py

```diff
@@ -344,19 +360,31 @@ def get_ves(comp, jc):
                     vbytmp.append(tsum[0])
                 tmpve['verifiedby'] = vbytmp
             # get the parent requirement
-            query = ("select cf.id, cf.cfname, cvf.textvalue from customfieldvalue cvf "
+            query = ("select cf.id, cf.cfname, cvf.textvalue, "
+                     "(select customvalue from customfieldoption where id = cvf.stringvalue) "
```

Comment:
Expressing this as an inner join might be more idiomatic and similar to the other joins below.  `cfo.customvalue [...] inner join customfieldoption cfo on cvf.stringvalue = cfo.id`